contrib/linode/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+volumes:
+  lbcd:
+    external: true
+
+services:
+  lbcd:
+    image: jeffreypicard/lbcd:deployment
```

Comment:
Did you mean to use a pre-built image, or we can use the one built from Dockerfile.deploy?